packages/poll/poll.0.1.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis:
+  "Portable OCaml interface to macOS/Linux/Windows native IO event notification mechanisms"
+description:
+  "poll provides a portable OCaml interface to IO event notification mechanisms on macOS, Linux and Windows. It uses kqueue on macOS, epoll on Linux, and uses a vendored copy of wepoll on Windows."
+maintainer: ["Anurag Soni <anurag@sonianurag.com>"]
+authors: ["Anurag Soni"]
+license: "MIT"
+tags: ["epoll" "kqueue" "wepoll"]
+homepage: "https://github.com/anuragsoni/poll"
+doc: "https://anuragsoni.github.io/poll"
+bug-reports: "https://github.com/anuragsoni/poll/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "base-unix"
+  "ppx_optcomp"
+  "kqueue" {>= "0.2.0"}
+  "dune-configurator"
+  "ppx_expect" {with-test}
+  "ocaml" {>= "4.12"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
```

Comment:
This really needs to be done in 2 steps?

Reply:
Sadly this is autogenerated by dune just in case the user uses the experimental `dune-site` plugin.
However with dune 3.0 this should go away: https://github.com/ocaml/dune/pull/4861